test/e2e/run.sh

```diff
@@ -23,27 +23,47 @@ export cni_plugin=${cni_plugin:-cilium}
 export cni_release=${cni_release:-latest}
 TOPOLOGY_DIR=${TOPOLOGY_DIR:=e2e}
 
-K8S_REPO="https://github.com/kubernetes/kubernetes"
+GH_K8S_REPO="kubernetes/kubernetes"
 export k8s_release=${k8s_release:-"latest"}
 export k8s_version=""
 
 source "$LIB_DIR"/vm.bash
 
+list-github-releases () {
+    local repo="$1" max="${max_releases:-20}" page=1 tags="" more="" sep=""
+    local total=0 cnt=-1
+
+    while [ "$cnt" != 0 -a "$total" -lt "$max" ]; do
+        echo -n "fetching releases for $repo (page $page)..." 1>&2
+        if more=$(wget -q "https://github.com/$repo/releases?page=$page" -O- | \
+                      grep -E '.*/releases/tag/v[0-9]+.[0-9]+.[0-9]+"' | \
+                      sed -E 's|.*/releases/tag/(v[0-9]+.[0-9]+.[0-9]+).*|\1|g')
+        then
+            tags="$tags$sep$more"
+            sep=" "
+            cnt="$(echo $more | wc -w)"
+            total=$((total+cnt))
+            echo " got $cnt more, $total in total..." 1>&2
+            let page=$page+1
```

Comment:
Style-nit: on line 45 (and elsewhere in the code) arithmetic evaluation and variable assignment happens with var=$(( ... )), so maybe 47 could be written similarly without using `let`.